src/components/CommunityPartners/CommunityPartners.vue

```diff
@@ -0,0 +1,260 @@
+          extraClass: "rounded-full",
+        },
+        {
+          name: "GitHub",
+          image: "https://github.com/GitHub.png",
+          link: "https://github.com",
+          description:
+            "GitHub is one of the world’s largest community of developers. It’s an intricate platform that fosters collaboration and communication between developers.",
+          extraClass: "rounded-full",
+        },
+        {
+          name: "Web3Community",
+          image: "https://github.com/Web3Community.png",
+          link: "https://web3community.github.io",
+          description:
+            "An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀",
+          extraClass: "rounded-full",
+        },
+        {
+          name: "Experify",
+          image: "/assets/experify.png",
+          link: "https://www.facebook.com/Experify3DPrint",
+          description:
+            "A 3D printing and 3D scanning service placed in Plovdiv. We provide wide variety of services and materials: prototyping, cosplay props, educational projects, robotics, etc.",
+          extraClass: "",
+        },
+        {
+          name: "Microsoft For Startups",
+          image: "/assets/microsoft.png",
+          link: "https://startups.microsoft.com/",
+          description:
+            "We believe that entrepreneurship should be open to all and that anyone with an idea deserves a way to innovate and grow.  Founders Hub helps remove traditional barriers to building a tech company by democratizing historically inaccessible resources including expert mentorship.",
+          extraClass: "",
+        },
+        {
+          name: "intlify",
+          image: "https://github.com/intlify.png",
+          link: "https://intlify.dev/",
+          description:
+            "Intlify is a project that aims to improve Developer Experience in software internationalization. We will aim to improve the Developer Experience of internationalization by providing libraries, frameworks, and tools that break down barriers to internationalization.",
+          extraClass: "rounded-full",
+        },
+        {
+          name: "Code for Japan",
+          image: "/assets/code_for_japan.jpg",
+          link: "https://www.code4japan.org/",
+          description:
+            "Code for Japan is one of the largest communities of civic hackers in Japan. We organize hackathons to improve and promote civic tech and collaborate with local governments and communities.",
+          extraClass: "rounded-full",
+        },
+      ],
+      isModalOpen: false,
+      isModalLoading: false,
+      isModalSuccessful: false,
+      isModalFailed: false,
+      sheetDbBaseUrl: 'https://sheetdb.io/api/v1/',
+      sheetDbConfig: {
+        address: import.meta.env.PUBLIC_SHEETDB_ADDRESS,
+        auth_login: import.meta.env.PUBLIC_SHEETDB_LOGIN,
+        auth_password: import.meta.env.PUBLIC_SHEETDB_PASSWORD,
```

Comment:
If you want to test this now, you must hard code this value. You can get it by creating a Google Form and creating an api with https://sheetdb.io. Don't forget to enable basic auth in the settings.

Reply:
Update: I've updated the version in this PR.

Reply:
I already created it, does the google sheet need to be public or is it okay if I make it private?

Reply:
It works whether it's public or private.

Reply:
> It works whether it's public or private.

Okay I set it to private

Reply:
I need to put this on the environment variable right?

- The spreadsheet URL
- auth login and auth password

or only auth login and pass?

Reply:
You need three things:
- address (a.k.a API ID)
- login
- password

The address can be found here: https://sheetdb.io/apis
![Screenshot 2021-12-20 072808](https://user-images.githubusercontent.com/28738855/146694912-c7f3747c-de8d-489d-8303-0e50ff5d2818.png)

The login and password can be found in the settings:
![Screenshot 2021-12-20 073011](https://user-images.githubusercontent.com/28738855/146694916-3e8d7327-2efd-4e8d-9b0b-8d64f3d9fb08.png)

Reply:
I should probably rename the variable `address` to `api_id` to avoid confusion.

Reply:
> Yes you need to put them in the environment variables.
> 
> You need three things:
> 
> * address (a.k.a API ID)
> * login
> * password

Cool thank you 🎉 

> I should probably rename the variable `address` to `api_id` to avoid confusion.

Hehehe yes

Reply:
> Update: I've updated the version in this PR.

Thanks but the tests fails due to the builds failing.

Reply:
CC @krishguptadev

Reply:
It should be on localhost:3000/devprotocol.xyz/community-partner due to the next update 👍

Reply:
Update: I've added a patch to make it appear or /community-partners

Reply:
Awesome 💯